obelix/idefix/trigger/dispatcher.go

```diff
@@ -0,0 +1,16 @@
+package trigger
+
+import (
+	"github.com/Shopify/sarama"
+	"idefix/operator"
+	"idefix/redis"
+	"reflect"
+)
+
+type Dispatcher interface {
+	Parse(data []byte) (reflect.Value, error)
+	LookForChange(op *operator.IdefixOperator, key, old string) error
+	SendToKafka(kp sarama.SyncProducer, workflowID string) error
+	GetRedisState(rc *redis.Client, key string) (string, error)
+	UpdateRedisState(rc *redis.Client, key, newer string) error
+}
```

Comment:
A couple of line of documentation would be nice

Reply:
9e9f34e5d856ee8112ba4132e897426538f9612d